tests/src/test/java/org/eclipse/hono/tests/registry/CredentialsManagementIT.java

```diff
@@ -81,10 +81,10 @@
         ORIG_BCRYPT_PWD = encoder.encode("thePassword");
     }
 
+    private final AtomicReference<String> resourceVersion = new AtomicReference<>();
```

Comment:
this would probably better be initialized as part of the `@BeforeEach` method ...

Reply:
Done.